### PR TITLE
Fc32

### PR DESCRIPTION
**Pull request by fepitre**
https://github.com/QubesOS/qubes-issues/issues/5763

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,12 @@
-sudo: required
-dist: bionic
-language: generic
-install: git clone https://github.com/QubesOS/qubes-builder ~/qubes-builder
+import:
+  - source: QubesOS/qubes-continuous-integration:R4.1/travis-base-r4.1.yml
 script: 
- - ~/qubes-builder/scripts/travis-build
```
Comment (marmarek) [inline .travis.yml:6]: A side effect of travis-build script was copying (actually - hardlinking) installer-qubes-os into ~/qubes-builder. See travis failure now.

**Review by marmarek — CHANGES_REQUESTED** [review]

**Comment by marmarek** [comment]
Travis build still fails. In a different way (losetup failed).

**Comment by JarrahG** [comment]
Not sure whether this is fixed elsewhere, but building the ISO fails with self.groups["qubes-ui"] not being defined. 
```python
-> Building installer-qubes-os iso for fc32 (logfile: build-logs/installer-qubes-os-iso-fc32.log)...
--> build failed!
    main(ns, persistdir, cachedir)
  File "/usr/lib/python3.8/site-packages/pungi/scripts/pungi_gather.py", line 163, in main
    packages, conditional_packages = ksparser.get_packages(dnf_obj)
  File "/usr/lib/python3.8/site-packages/pungi/ks.py", line 160, in get_packages
    ) = dnf_obj.comps_wrapper.get_packages_from_group(
  File "/usr/lib/python3.8/site-packages/pungi/dnf_wrapper.py", line 97, in get_packages_from_group
    group = self.groups[group_id]
KeyError: 'qubes-ui'
make[1]: *** [Makefile:87: iso-installer-gather] Error 1
```

**Comment by marmarek** [comment]
> but building the ISO fails with self.groups["qubes-ui"] not being defined.

This looks like submodules not updated. Call `git submodule update --init` in installer-qubes-os dir.

**Comment by fepitre** [comment]
> git submodule update --init

It does not change anything.

**Pull request merged**